blog/_posts/2018-09-17-Code-2-a-good-summer.md

```diff
@@ -0,0 +1,138 @@
+---
+title: Team GirlsCode MK - Code 2 a good summer
+layout: post
+created_at: Sat Sep 15 2018
+permalink: blog/2018-07-25-Code-2-a-good-summer
```

Comment:
Hi @gtl90 ! Thanks for your PR. Lovely blog post. Please check the bunch of minor comments I have. 

Filename and permalink needs to be the same.